### ISSUE TITLE
About evaluation index calculation

### ISSUE THREAD

**Issue by PANHAOC** [Luffy03/VoCo#22]
Hello, this model is a 3D model, so when verifying the test set, you can directly input the 3D image to get the segmentation result. I would like to ask, if I am a 2D model, facing the verification set of 3D images, there is no unified slicing standard, how should I slice the 3D image and then calculate the dice index of the model on this data set?

**Comment by Luffy03** [comment]
Many thanks for your attention to our work!  But sorry, I cannot understand your problem very clearly.
1. What do you mean no unified slicing standard? For 2d slices, can you input each slice to the model one by one?
2. Dice calculation is not related to 2D or 3D. And what is your meaning of dice index? The dice of each class?

**Comment by PANHAOC** [comment]
> Many thanks for your attention to our work! But sorry, I cannot understand your problem very clearly.
> 
> 1. What do you mean no unified slicing standard? For 2d slices, can you input each slice to the model one by one?
> 2. Dice calculation is not related to 2D or 3D. And what is your meaning of dice index? The dice of each class?

What I mean is that when a 2D model is trained on medical images, it is necessary to pre-process the medical images and select some appropriate slices for training. For the test, when calculating the final dice, do you also select some slices to calculate the dice (this will result in different slices selected by each person, and the dice will also be different), or do you select all slices of each sample in the test to participate in the dice calculation?

**Comment by Luffy03** [comment]
Oh, for 2D models, you should select all slices for metric calculation, otherwise the comparisons would be not fair. 

By the way, our repo will not support 2D models since we find that 3D models are better. For 2d pre-training, we find that DINO transformers can already get good performances, but still worse than 3D models. For 2d medical segmentation, I think nnunet is a better choice, which provide clean and powerful codebase.

**Comment by PANHAOC** [comment]
Thank you very much for your reply！

**Comment by Luffy03** [comment]
Dear researchers, our work is now available at [**Large-Scale-Medical**](https://github.com/Luffy03/Large-Scale-Medical), if you are still interested in this topic. Thank you very much for your attention to our work, it does encourage me a lot!
